composer.json

```diff
@@ -29,6 +29,7 @@
     }
     , "require-dev": {
         "phpunit/phpunit": "4.1.*"
+        , "cakephp/cakephp-codesniffer": "2.*"
```

Comment:
I can understand the reasoning behind it, but it's as useful to the human mind as yoda conditions :P